### DISCUSSION TITLE
Squircle Does not seem to be working properly

### DISCUSSION THREAD

**Discussion by K4TieK** [ramensoftware/windhawk-mods#881] [General]
![Screenshot 2024-08-19 184027](https://github.com/user-attachments/assets/a352a0c1-a5ee-4479-b35a-2f932f78efd2)

![image](https://github.com/user-attachments/assets/2fbb9a8f-2451-457f-87c8-ba5a11539fa4)

**Comment by m417z** [comment]
It's best to write about it [here](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues) and tag [the mod author](https://github.com/AsvnDG).
